libs/go-libs/otlp/cli.go

```diff
@@ -0,0 +1,23 @@
+package otlp
+
+import (
+	"sync"
+
+	flag "github.com/spf13/pflag"
+)
+
+var (
+	once sync.Once
+)
+
+const (
+	OtelResourceAttributes = "otel-resource-attributes"
+	OtelServiceName        = "otel-service-name"
+)
+
+func InitOTLPFlags(flags *flag.FlagSet) {
+	once.Do(func() {
```

Comment:
why use "once" ?
It is the responsability of the caller to not call this method multiple times.
Smell like defensive programming ;)

Reply:
Ok, found why!